install/installer/cluster_dev.py

```diff
@@ -576,7 +576,11 @@ def create_aws_user_and_permissions(
     # AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
     try:  # Required "iam:ListUsers"???, "iam:ListAccessKeys"
         response = iam.list_access_keys(UserName=user)
-
+    except iam.exceptions.NoSuchEntityException:  # User not yet exist
+        pass
+    except iam.exceptions.ClientError as error:
+        sys.exit(f'\nERROR: {error}\n\nRights what you need described here: \n{link}')
```

Comment:
Introduce a special return code for such types of failures to make it easier to debug the behavior:
```suggestion
        logger.error(
            'ERROR: %s. The required privileges are described at %s',
            error, link,
        )
        sys.exit(3)
```

Reply:
I have 5 same message like this. Should I use unique return code in each case?

Reply:
No, use one code per a unique type of problem. You can also have some Enum with different return codes for different situations. Like
```python
class ReturnCode(IntEnum):
    SUCCESS = 0
    NETWORK_ERROR = 2
    USER_ERROR = 3
```

You can also `import errno` to reuse some of the common predefined codes from the list @ https://man7.org/linux/man-pages/man3/errno.3.html if they match your use case.

Reply:
No idea how error codes can help in debug. 
Also, errors provided by boto3 different than specified in [errno](https://docs.python.org/3/library/errno.html). So I can't use [something like this](https://christophergs.com/python/2017/01/15/python-errno/).

But globally, it's very good suggestion for separate many errors catches by one `except`, thanks.

Reply:
This has nothing to do with boto3 codes. You should define your own codes and use them consistently across this script. I just meant that you could reuse some more commonly known values:

```python
class ReturnCode(IntEnum):
    SUCCESS = 0
    NETWORK_ERROR = errno.ENETUNREACH
    USER_ERROR = errno.ENOTSUP
```